wlauto/workloads/manual/__init__.py

```diff
@@ -58,6 +60,9 @@ class ManualWorkload(Workload):
         Parameter('view', default='SurfaceView',
                   description="""Specifies the View of the workload. This enables instruments that require a
                                  View to be specified, such as the ``fps`` instrument."""),
+        Parameter('package',
+                  description="""Specifies the package name of the workload. This enables instruments that
+                                 require a Package to be specified, such as the ``fps`` instrument."""),
```

Comment:
It's probably worth updating the description for this and for `'view'` above explaining that one is only required for "surfaceflinger" (pre- API level 23), and the  other only for "gfxinfo" (post- API level 23) methods of fps collection.